interoperability_report.py

```diff
@@ -559,13 +581,13 @@ def parser():
             metavar='test_suite_dictionary_file',
             type=str,
             help='Test Suite that is going to be tested. \
-                Test Suite is a file with a dictionary defined, it should \
+                Test Suite is a file with a Python dictionary defined. It should \
                 be located on the same directory as interoperability_report. \
-                By default is test_suite. \
-                To call it do not write ".py", only the name of the \
-                file. \
+                This value should not contain the extension ".py", \
+                only the name of the file. \
                 It will run all the dictionaries defined in the file. \
-                Multiple files can be passed.')
+                Multiple files can be passed. \
```

Comment:
how do we separate multiple files? or do we need to add several times the same option? And, in case we pass 2 files, does the script test both of them? how about the parameters? (ignore_test or select the test to run) do they apply to both test suites?

Reply:
to call the script: -s test_suite_1 test_suite_2
yes, it tests both of them
in case of ignore_test, select_test: if the tests exist in both test suites there is not any problem. If the tests exist in one test suite and not in the other, it will test the test suites in order, so if the test case that does not exist is in the first test suite it will exit the program, if it is in the second test suite it will test the first test suite and then exit the program

Reply:
we decided to only allow one test suite